node/Cargo.toml

```diff
@@ -24,50 +24,50 @@ serde_json = "1.0.61"
 structopt = '0.3.8'
 
 # local dependencies
-hydra-dx-runtime = {path = '../runtime', version = '2.0.0'}
-module-amm-rpc = {path = '../pallets/amm/rpc', version = '2.0.0'}
-primitives = {path = '../primitives', version = '2.0.0'}
+hydra-dx-runtime = {path = '../runtime', version = '3.0.0'}
```

Comment:
do we even need to version these ourselves?

Reply:
yeah, it shouldn't be necessary

Reply:
no, not necessary. it is already done for some of the local dependencies.

Reply:
done for all local dependencies